app/Commands/EnableCommand.php

```diff
@@ -47,7 +59,20 @@ public function enableableServices(): array
         })->toArray();
     }
 
-    public function enable(string $service): void
+    public function enableableServicesByCategory(): array
```

Comment:
One last request! Can we sort the list after we're done so the category list is alphabetized?

Reply:
Category sorted. 💪